client/state/site-settings/actions.js

```diff
@@ -68,6 +68,7 @@ export function requestSiteSettings( siteId ) {
 				};
 
 				dispatch( receiveSiteSettings( siteId, savedSettings ) );
+				dispatch( receiveSiteOptions( siteId, savedSettings ) );
```

Comment:
1. So that the "concerns" are "symmetric" here -- settings and options.
2. Maybe in the future people will include other options; this way they can just add it inside `receiveSiteOptions()`

😄 What do you think?

Reply:
👍